js/views/modals/listingDetail/Listing.js

```diff
@@ -90,6 +92,29 @@ export default class extends BaseModal {
       this.listenTo(app.localSettings, 'change:bitcoinUnit', () => this.showDataChangedMessage());
     }
 
+    this.rating = this.createChild(Rating);
+
+    // get the ratings data, if any
+    this.ratingsFetch =
+      $.get(app.getServerUrl(`ob/ratings/${this.vendor.peerID}/${this.model.get('slug')}`))
+        .done(data => this.onRatings(data))
+        .fail((jqXhr) => {
+          if (jqXhr.statusText === 'abort') return;
+          // if there are no ratings, no response is returned.
+          if (jqXhr.status === 200) {
```

Comment:
Is this if block really needed? I'm don't think there would ever be a case where a 200 triggers the fail handler.